VulkanHppGenerator.cpp

```diff
@@ -13957,6 +14062,21 @@ void VulkanHppGenerator::readRequireEnum(
             m_constants[name] = { type, value, line };
           }
         }
+
+        // if we have a name and value, insert to enumConstants map
+        if ( attributes.contains( "name" ) && attributes.contains( "value" ) )
```

Comment:
At this point, it's already checked that there is a "value".
I think, you could just add that unconditionally, as "name" was checked as required above.
Maybe you could add a check like
```
        checkForError(
          !supported || name.ends_with( "_EXTENSION_NAME" ) || name.ends_with( "_SPEC_VERSION" ), line, "encountered unexpected enum <" + name + ">" );
```